source/common/modules/markdown-editor/hooks/note-preview.js

```diff
@@ -0,0 +1,100 @@
+const tippy = require('tippy.js').default
+const { ipcRenderer } = require('electron')
+
+/**
+ * A hook for displaying link tooltips which can be used to visually
+ * enable users to click a link (without having to press down Ctrl/Cmd)
+ *
+ * @param   {CodeMirror}  cm  The instance to attach to
+ */
+module.exports = (cm) => {
+  cm.getWrapperElement().addEventListener('mousemove', (event) => {
+    let a = event.target
+
+    // Only for note links
+    if (!a.classList.contains('cm-zkn-link')) {
+      return
+    }
+
+    // If there's already a tippy on the instance, don't re-render it
+    if (a.hasOwnProperty('_tippy')) {
+      return
+    }
+
+    // Initialise displayed attributes to 'loading'
+    let title = 'loading'
```

Comment:
Please add a comment `// TODO: Translate!` after any one of these lines, just so that I can perform a big, broad search for all of the TODOs shortly before releasing 2.0 so I don't forget to translate those strings (the important part is the all-uppercase "TODO" keyword)